plugins/inventory/flightctl.py

```
@@ -0,0 +1,592 @@
+            except ValueError:
+                a = date_parser.isoparse(actual)
+                b = date_parser.isoparse(val)
+
+        # for in/notin operators, val should eval() to list
+        if op in ('in', 'notin'):
+            try:
+                val_list = ast.literal_eval(val)
+                if not isinstance(val_list, list):
+                    raise ValueError()
+            except Exception:
+                raise ValidationException(f"Invalid list syntax in selector: {sel!r}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)